dist/docs/guide/customization/promote-customization.adoc

```diff
@@ -0,0 +1,44 @@
+### Promoting custom artifacts
+
+The fist step to apply changes from a bundle to a server, is to promote the customized artifacts into the custom channel.
+
+Promoting is a two-step process:
+
+ . all artifacts from a bundle are deployed in the custom Maven repository.
+ . update the channel file with new versions of customized artifacts and deploy it in the repository with incremented version.
+
+#### Register custom channel
+*Single server channel*
+
+```
+prospero channel initialize --dir=<PATH/TO/SERVER>
```

Comment:
This would initialize an empty channel. Does a user ever need to do that? Does it need to be done before `prospero channel promote --archive=<bundle.zip> --dir=<PATH/TO/SERVER>`? If not, maybe user doesn't need to know about this. (Or couldn't it be done automatically when it's needed?)

Reply:
This is needed to setup the server - register the new channel and repository in prospero-config.yaml. In single-server scenario it could be done when first `promote` is called as long as user is happy with defaults.